vali_objects/vali_dataclasses/perf_ledger.py

```diff
@@ -715,20 +725,14 @@ def condense_positions(self, tp_to_historical_positions: dict[str: Position]) ->
     def build_perf_ledger(self, perf_ledger: PerfLedgerData, tp_to_historical_positions: dict[str: Position], start_time_ms, end_time_ms, miner_hotkey, realtime_position_to_pop) -> bool:
         #print(f"Building perf ledger for {miner_hotkey} from {start_time_ms} to {end_time_ms} ({(end_time_ms - start_time_ms) // 1000} s)")
         if len(perf_ledger.cps) == 0:
-            perf_ledger.init_with_first_order(end_time_ms, point_in_time_dd=1.0)
+            perf_ledger.init_with_first_order(end_time_ms, point_in_time_dd=1.0, current_portfolio_value=1.0,
```

Comment:
If it is truly the first order then it will get set to 1.0 for the attributes. Sometimes the first order in the perf ledger happens many orders after the first true order. That is because the trader has been active for more than 90 days. That is the behavior that is being fixed.

Reply:
See this miner for an example of the bug https://dashboard.taoshi.io/miner/5CALivVcJBTjYJFMsAkqhppQgq5U2PYW4HejCajHMvTMUgkC